core/templates/services/exploration-features.service.ts

```diff
@@ -45,12 +45,22 @@ export interface ParamChanges {
   providedIn: 'root'
 })
 export class ExplorationFeaturesService {
+  /**
+   * Static type variable to initialize a service
+   * by setting it to true or false.
```

Comment:
Variable to keep track of whether the service has been initialised.

Reply:
Updated 👍